src/null-safety/migrating-to-null-safety/index.md

```diff
@@ -0,0 +1,435 @@
+---
+title: Migrating to null safety
+description: How to move your existing Dart code to the world of null safety
+---
+
+**TODO: Null safety hasn't shipped yet and this doc still contains some TODOs
+for aspects of null safety (like the shipping stable version number) that aren't
+known yet. Fix those before landing this on master.**
+
+When we decided to add null safety to Dart, we wanted to get as much value out
```

Comment:
Consider linking to the main null safety topic here on the first occurrence of 'null safety'

Reply:
Done.